cli/dcoscli/service/main.py

```diff
@@ -305,4 +306,8 @@ def _log_marathon(follow, lines, ssh_config_file):
 
     emitter.publish(DefaultError("Running `{}`".format(cmd)))
 
+    # In the tests, we SIGKILL this process group, and the output
+    # doesn't get flushed unless we explicitly do so here.
+    sys.stderr.flush()
```

Comment:
Looks like python >3.3 added a flush argument for the `print` function. I say we move this flush into the emitter and just call flush every time we print. What do you think? This pattern breaks the abstraction and assumes that `emitter.publish(Error(...))` prints to `sys.stderr`